src/uci.cpp

```diff
@@ -376,8 +376,8 @@ string UCI::move(Move m, bool chess960) {
 
 Move UCI::to_move(const Position& pos, string& str) {
 
-  if (str.length() == 5) // Junior could send promotion piece in uppercase
-      str[4] = char(tolower(str[4]));
+  if (str.length() == 5)
+      str[4] = char(tolower(str[4])); // Lowercase the promotion piece character
```

Comment:
I think the important info is that some GUIs (probably PC Chess Junior) send uppercase promotion piece info, contrary to the standard, but we take care.